### ISSUE TITLE
Relevant IPR?

### ISSUE THREAD

**Issue by rossjones** [ckan/cla#3]
```
**You grant to the Open Knowledge Foundation a perpetual, non-exclusive,
irrevocable, worldwide, royalty-free, transferable license** (with the power to
sub-license) of all of your relevant intellectual property rights (including
```

This is unclear, what does 'relevant' mean in this context? Work I might do on a competing product (for instance) might be seen as relevant.

**Comment by rufuspollock** [comment]
Do you have a better phrasing? One clearly cannot drop "relevant" as that would widen the inclusion.

**Comment by rossjones** [comment]
I think it's already a bit wide, someone could argue relevant relates to work on Open Data Portals generally and people contributing to more than one might not like it.

How about something like...

```
You grant to the Open Knowledge Foundation a perpetual, non-exclusive,
irrevocable, worldwide, royalty-free, transferable license** (with the power to
sub-license) to all intellectual property rights on your contributions to CKAN
```

**Comment by davidread** [comment]
I agree this needs tidying up as 'relevant IPR' is too woolly. It might start to include comments on a CKAN website, or data uploaded to it.

I guess you if you say "contributions to CKAN" you'd usefully define CKAN to be the main code base (including documentation) and (official) extension code? I guess we're less concerned

Or maybe tackle it along the lines of "Relevant intellectual property is limited to source code and files that are communicated publicly (or with explicit consent) to the CKAN Association, such as comments, commits or pull requests on the official source repository, IRC and email groups."

**Comment by davidread** [comment]
Can we agree on the scope of the IPR captured by the licence? My guess:
- ckan code
- ckan docs (in the codebase)
- "core" extensions in /ckan/ckan
- "official"? extensions in /ckan/ckanext-*

I assume it excludes:
- extensions in /elsewhere - probably not
- content at ckan.org
- comments and package contributions to datahub.io etc

Could this be included in the CLA in some form?

**Comment by rufuspollock** [comment]
To be clear this CLA would be used in context of a pull request, and, where retrospective, would be focused on some specific contribution.

I therefore don't think we need something explicit in the CLA - rather the CLA will be used in a given context that makes its application clear.
